app/services/allocation/AssessorAllocationService.scala

```diff
@@ -56,27 +75,75 @@ trait AssessorAllocationService {
     }
   }
 
-  def allocateCandidates(newAllocations: command.CandidateAllocations): Future[Unit] = {
-    getCandidateAllocations(newAllocations.eventId).flatMap { existingAllocation =>
-      existingAllocation.allocations match {
-        case Nil => candidateAllocationRepo.save(persisted.CandidateAllocation.fromCommand(newAllocations)).map(_ => ())
-        case _ => updateExistingAllocations(existingAllocation, newAllocations).map(_ => ())
+  private val dateFormat = "dd MMMM YYYY"
+  private val timeFormat = "HH:mma"
+
+
+  def allocateCandidates(newAllocations: command.CandidateAllocations)(implicit hc: HeaderCarrier, rh: RequestHeader): Future[Unit] = {
```

Comment:
Well, yes, but:
- out of scope of this task.
- taking into account current test coverage I would prefer not to do unrelated refactorings.